ArduCopter/mode_auto.cpp

```diff
@@ -177,11 +177,26 @@ bool ModeAuto::allows_weathervaning() const
 // Go straight to landing sequence via DO_LAND_START, if succeeds pretend to be Auto RTL mode
 bool ModeAuto::jump_to_landing_sequence_auto_RTL(ModeReason reason)
 {
-    if (mission.jump_to_landing_sequence()) {
+    // Use stopping point as location for start of auto RTL
+    Vector3p stopping_point_NEU;
+    copter.pos_control->get_stopping_point_xy_cm(stopping_point_NEU.xy());
+    copter.pos_control->get_stopping_point_z_cm(stopping_point_NEU.z);
+    Location stopping_point {stopping_point_NEU, Location::AltFrame::ABOVE_HOME};
+
+    if ( ((g2.auto_rtl_type == 3) && mission.jump_to_shortest_landing_sequence(stopping_point)) ||
+         ((g2.auto_rtl_type == 4) && mission.jump_to_closest_mission_leg(stopping_point)) ||
+         ((g2.auto_rtl_type == 5) && mission.jump_to_shortest_mission_leg(stopping_point)) || 
```

Comment:
We could ommit the `jump_to_shortest_mission_leg` to make the patch smaller, I don't think your using it?